backend/dining/api_wrapper.py

```diff
@@ -125,6 +126,82 @@ def get_venues(self):
             results.append(value)
         return results
 
+    def load_menu(self, date=timezone.now().date()):
+        """
+        Loads the weeks menu starting from today
+        NOTE: This method should only be used in load_next_menu.py, which is
+        run based on a cron job every day
+        """
+
+        # Venues without a menu should not be parsed
+        skipped_venues = [747, 1163, 1731, 1732, 1733, 1464004, 1464009]
+
+        # TODO: Handle API responses during empty menus (holidays)
+        menu_base = OPEN_DATA_ENDPOINTS["MENUS"]
+        venues = Venue.objects.all()
+        for venue in venues:
+            if venue.venue_id in skipped_venues:
+                continue
+            response = self.request("GET", f"{menu_base}?cafe={venue.venue_id}&date={date}").json()
+
+            # Load new items into database
+            # TODO: There is something called a "goitem" for venues like English House.
+            # We are currently not loading them in
+            self.load_items(response["menus"]["items"])
+
+            menu = response["menus"]["days"][0]
+            dayparts = menu["cafes"][str(venue.venue_id)]["dayparts"][0]
+            for daypart in dayparts:
+                # Parse the dates in data
+                for time in ["starttime", "endtime"]:
+                    daypart[time] = make_aware(
+                        datetime.datetime.strptime(
+                            menu["date"] + "T" + daypart[time], "%Y-%m-%dT%H:%M"
+                        )
+                    )
+                dining_menu = DiningMenu.objects.create(
+                    venue=venue,
+                    date=menu["date"],
+                    start_time=daypart["starttime"],
+                    end_time=daypart["endtime"],
+                    service=daypart["label"],
+                )
+                # Append stations to dining menu
+                stations = self.load_stations(daypart["stations"])
+                dining_menu.stations.add(*stations)
+                dining_menu.save()
+
+    def load_stations(self, station_response):
+        # Store stations into list
+        stations = list()
+        for station_data in station_response:
+            # TODO: This is inefficient for venues such as Houston Market
+            station = DiningStation.objects.create(name=station_data["label"])
+            item_ids = [int(item) for item in station_data["items"]]
+            # Bulk add the items into the station
+            items = DiningItem.objects.filter(item_id__in=item_ids)
+            station.items.add(*items)
+            station.save()
+            stations.append(station)
+        # NOTE: use generator here?
```

Comment:
unnecessary to use a generator here since you're already passing it in as parameters to the stations on line 171 (which will then be bulk added by Django)